### DISCUSSION TITLE
Interpretation of proportions

### DISCUSSION THREAD

**Discussion by cjaggy** [neherlab/covid19_scenarios#670] [General] [answered]
When running the simulation I was not sure on how to interprete the figures reported under proportions. Is a fatal case also counted as critical or severe case? Similarly, is a critical case also counted as severe case? Or is an individual that runs through this state model only counted in the worst state the individual achieves?
Thanks for an answer.

**Comment by rneher** [comment, marked as answer]
These numbers sum to 100%. Critical is hence the fraction of people that fall critically ill and didn't die. Severe are those that get severely ill but don't need critical care etc...

**Reply by cjaggy** [reply]
Thanks for the feedback and sorry for the simple question. Not knowing the programming code, I was not sure on how the model keeps track on an individual moving along the status path and thereby avoiding multiple counts. But as the individual becomes "fractional" along the path I assume only the fraction that achieved the worst status is counted in the proportions.

**Reply by rneher** [reply]
not worries. we have gone back and forth on how the count this and settled on counting the most serious outcome for a case. We'll add this to the FAQ.

**Reply by cjaggy** [reply]
Great - this helps me  to have a better understanding  particularly also when comparing with the year to date figures from corona-data
